### ISSUE TITLE
Error 500 when editing an entity

### ISSUE THREAD

**Issue by tomchop** [yeti-platform/yeti#423]
This is the stacktrace:

```
May 21 14:41:11 Yeti uwsgi[1071]: ERROR:app:Exception on /entity/edit/5bcd3b23f8e3920703e4f095 [POST]
May 21 14:41:11 Yeti uwsgi[1071]: Traceback (most recent call last):
May 21 14:41:11 Yeti uwsgi[1071]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 2292, in wsgi_app
May 21 14:41:11 Yeti uwsgi[1071]:     response = self.full_dispatch_request()
May 21 14:41:11 Yeti uwsgi[1071]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1815, in full_dispatch_request
May 21 14:41:11 Yeti uwsgi[1071]:     rv = self.handle_user_exception(e)
May 21 14:41:11 Yeti uwsgi[1071]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1718, in handle_user_exception
May 21 14:41:11 Yeti uwsgi[1071]:     reraise(exc_type, exc_value, tb)
May 21 14:41:11 Yeti uwsgi[1071]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1813, in full_dispatch_request
May 21 14:41:11 Yeti uwsgi[1071]:     rv = self.dispatch_request()
May 21 14:41:11 Yeti uwsgi[1071]:   File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1799, in dispatch_request
May 21 14:41:11 Yeti uwsgi[1071]:     return self.view_functions[rule.endpoint](**req.view_args)
May 21 14:41:11 Yeti uwsgi[1071]:   File "/usr/local/lib/python2.7/dist-packages/flask_classy.py", line 200, in proxy
May 21 14:41:11 Yeti uwsgi[1071]:     response = view(**request.view_args)
May 21 14:41:11 Yeti uwsgi[1071]:   File "./core/web/helpers.py", line 37, in inner
May 21 14:41:11 Yeti uwsgi[1071]:     return f(*args, **kwargs)
May 21 14:41:11 Yeti uwsgi[1071]:   File "./core/web/frontend/generic.py", line 71, in edit
May 21 14:41:11 Yeti uwsgi[1071]:     return self.handle_form(id=id)
May 21 14:41:11 Yeti uwsgi[1071]:   File "./core/web/frontend/generic.py", line 105, in handle_form
May 21 14:41:11 Yeti uwsgi[1071]:     obj = klass.objects.get(id=id)
May 21 14:41:11 Yeti uwsgi[1071]: AttributeError: 'NoneType' object has no attribute 'objects'
May 21 14:41:11 Yeti uwsgi[1071]: [pid: 1760|app: 0|req: 13/55] 192.168.180.1 () {54 vars in 1242 bytes} [Tue May 21 14:41:11 2019] POST /entity/edit/5bcd3b23f8e3920703e4f095 => generated 291 bytes in 9 msecs (HTTP/1.1 500) 2 headers in 84 bytes (1 switches on core 0)
```

Likely related to the recent change introduced in `frontend/generic.py`

**Comment by phutch11** [comment]
Thanks for looking into this.

**Comment by tomchop** [comment]
Fixed in #430. Thanks for reporting!
